components/batch/default/index.mjs

```diff
@@ -109,30 +128,42 @@ export const mainAsync = async (process, configuration) => {
     const { tokens } = configuration.command;
     const command = await compileConfigurationCommandAsync(configuration, env);
     logDebug("spawn child command = %j", command);
-    const { signal, status } = await spawnWithHandlerAsync(
-      command,
-      children,
-      tokens,
-      false,
-    );
+    let signal = null;
+    let status = 0;
+    let done = false;
+    let timer = null;
+    const flush = async () => {
+      timer = null;
+      if (!done) {
+        await flushBackendSessionAsync(backend, session, false);
+        timer = setTimeout(flush, FLUSH_TIMEOUT);
```

Comment:
You can also do `timer.refresh()` instead.

Reply:
Done, thanks!